scripts/android-tracy-runnable-tutorial.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
```

Comment:
nit: replace `-` with `_` in file name to match the other scripts in https://github.com/google/iree/tree/main/scripts?

Reply:
done